content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -185,4 +175,23 @@ Insurance | Do you have insurance? | 1 Yes
 Mobile_money_classification |   | 0 no mobile money and no other financial service (saving, borrowing, insurance)
 |       |               | 1 no mobile money, but at least one other financial service
 |       |               | 2 mobile money only
-|       |               | 3 mobile money and at least one other financial service
\ No newline at end of file
+|       |               | 3 mobile money and at least one other financial service
+
+
+## Instructions:
+
+1. Import the required libraries and load the datadatset [training](training.csv).
+   
+2. Explore the dataset. Are there any missing observations or columns where the data do not seem valid?
+
+3. Get basic descriptive statistics for the dataset.
+
+4. Create appropriate graphs to visually represent the relationship between financial services accessed (non-mobile, mobile, both) and age, gender, marital status, land ownership and type of income. Provide an interpretation for each graph.  
+
+5. What can you conclude about use of financial services in Tanzania so far? Which demographic are associated with mobile money use ?
+
+## Supporting Material:
+
+1. [Data Visualization Techniques](https://www.datacamp.com/blog/data-visualization-techniques)
```

Comment:
Headings need to be consistent. Why do some things have colons and others dont? 
Rather dont put in colons. 

Also, why is this in a list? It looks like there is only one item in the list. That's a bit strange looking to me.